.github/workflows/gradle.yml

```
@@ -20,8 +20,16 @@ jobs:
     - name: Set up JDK
       uses: actions/setup-java@v2
       with:
-        distribution: 'adopt'
+        distribution: 'temurin'
         java-version: '8'
+    - name: Check plugin
+      run: ./gradlew check
+    - name: Publish Test Report
+      if: always()
```

This is redundant, isn't it?

---

Why? Before this change tests weren't executed on CI.

---

My bad, I thought `always()` was the default, but in reality it is `success()`.

BTW, according to [the docs](https://docs.github.com/en/actions/learn-github-actions/expressions#always), it seems it should be:

```suggestion
      if: ${{ always() }}
```

---

Interesting, we use that notation in several places I think